src/Core/src/Platform/iOS/ContentView.cs

```diff
@@ -116,6 +116,13 @@ void SetClip()
 			if (Subviews.Length == 0)
 				return;
 
+			var child = Subviews[0];
+
+			if (child is MauiScrollView mauiScrollView)
+			{
+				return;
```

Comment:
Without use paddings, the same example:
![image](https://user-images.githubusercontent.com/6755973/226651867-f4cc44b0-30d8-457e-a313-769e3862af41.png)

Will not be clipped on iOS/macOS by the Border shape, and the UIScrollView will overlay the round corners of the Border. I understand that avoiding the scroll is an important issue, but maybe we have another option. Wrapping the ScrollView if it is a child of a Border and clipping the wrapper instead the ScrollView?

Reply:
![Simulator Screen Shot - iPhone 14 Pro Max - 2023-03-21 at 15 36 36](https://user-images.githubusercontent.com/1235097/226658657-d99413b4-71c4-4b8f-bcb3-d0ddfd5b4be5.png)

ok on iOS looks like this